mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -189,7 +189,7 @@
       expect(@source_page.format.render_titled).to eq(@source_page.format.render_titled_with_voting)
     end
 
-    it "renders open view with :custom_source_header to be true" do 
+    it "renders open view with :custom_source_header to be true" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.